src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/changes/benificialowner/BeneficialOwnerChange.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.benificialowner;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.Change;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.benificialowner.psc.Psc;
+
+public abstract class BeneficialOwnerChange<T extends Psc> extends Change {
+  @JsonProperty("change")
+  private String change = "pscAppointmentChange";
+
+  @JsonProperty("appointmentId")
+  private String appointmentId;
+
+  @JsonProperty("psc")
+  private T psc;
+
+  @JsonProperty("pscPreviousDetails")
```

Comment:
Is this needed? It's not mentioned in the mapping documentation

Reply:
Sorry thought I deleted this